index.js

```diff
@@ -0,0 +1,17 @@
+const util = require('util'),
```

Comment:
Will this be used for back end repos as well? Can you write it in ES5?

From the architecture invariants:
> We write full-stack libraries, back-end applications, and back-end libraries in ES5.

That conversion work is in the backlog, but hasn't been picked up yet. Platform is focused on another engineering investment right now, which is guts integration tests which do not need core. Maybe this could be next.